### ISSUE TITLE
关于GlobalPointer的一点疑惑，还望解答

### ISSUE THREAD

**Issue by liyunhan** [zhengyanzhao1997/TF-NLP-model#14]
您好，非常有幸在CSDN看到了您的博文，关系抽取系列文章阅读下来后对您是无比的敬佩（尤其是附有pytorch和tf的实现）

但是有点关于GlobalPointer的小问题想要请教，相信您关于GlobalPointer的代码实现也参考了苏神的源码，我看到苏神在GlobalPointer的相关实现中用到了sequence_mask，即` logits = sequence_masking(logits, mask, '-inf', 2)
        logits = sequence_masking(logits, mask, '-inf', 3)` 但是在调用GlobalPointer的时候却没有传入mask矩阵，即`model = build_transformer_model(config_path, checkpoint_path)
output = GlobalPointer(len(categories), 64)(model.output)`请问这是为何？

**Comment by zhengyanzhao1997** [comment]
我理解model = build_transformer_model(config_path, checkpoint_path)的model.output中包含了output_embedding 和 mask
